### DISCUSSION TITLE
Modules entities cannot be activated. Component 'AtomViewportDisplayInfoSystemComponent'...

### DISCUSSION THREAD

**Discussion by nzo3d** [o3de/o3de#16948] [Help Wanted]
Hello, I can't use O3DE. I have already installed/uninstalled several times, but I still get the same error when opening a project.

Modules entities cannot be activated.
Component 'AtomViewportDisplayInfoSystemComponent' in missing another required service: 0xf2add773

I tried disabling the firewall and antivirus, but I still get the same error message.

Can you help me resolve this problem?

**Comment by youraveragementor** [comment]
have you fixed the problem

**Comment by youraveragementor** [comment]
by the way, i just go it lol hoping to make the next AAA but this....

**Comment by PhilippTemkin** [comment]
Same thing :( How do I fix this? It comes out of the box. I just completed a fresh install for the first time without any warnings and this is what I sowed when I tried to run the lunch blank project -_-

**Reply by Andre-LA** [reply]
Did you [build the project](https://www.docs.o3de.org/docs/welcome-guide/create/creating-projects-using-project-manager/#build-the-o3de-project) before opening it? IIRC this error shows when the project isn't built yet.
